src/css/style.css

```diff
@@ -769,7 +705,7 @@ input[type="checkbox"], input[type="radio"] {
 }
 
 .login-input {
-    font-family: Montserrat-Bold;
+    font-weight: bolder;
```

Comment:
Gotta say, this looks horrible on my laptop 😅
But the login page could be refined in the future

Reply:
What broweser?